config.json

```diff
@@ -24,6 +24,14 @@
                         "s3-website-sa-east-1.amazonaws.com",
                         "s3-website.localhost",
                         "s3-website.scality.test"],
+    "replicationEndpoints": [{
```

Comment:
why doesn't "zenko" site have a type?

Reply:
Endpoints that are replicating to a Zenko destination do not have an external backend "type".